resident/resident-service/src/main/java/io/mosip/resident/constant/ResidentErrorCode.java

```diff
@@ -113,7 +113,10 @@ public enum ResidentErrorCode {
 	OTP_REQUEST_FLOODED("RES_SER_510", "Innumerous OTP requests received"),
 	EID_NOT_BELONG_TO_SESSION("RES-SER-511", "The entered EID is not associated with the UIN/VID used to log in."),
 	GRIEVANCE_TICKET_GENERATION_FAILED("RES-SER-512", "Unable to Generate Ticket of grievance"),
-	MISSING_INPUT_PARAMETER("RES-SER-414", "Missing input Parameter- %s");
+	MISSING_INPUT_PARAMETER("RES-SER-414", "Missing input Parameter- %s"),
+	VID_CREATION_FAILED("RES-SER-513",
```

Comment:
Correction to the enum constant: VID_CREATION_FAILED_WITH_REVOCATION

Reply:
Done.